doc/source/using-ray.rst

```diff
@@ -13,7 +13,7 @@ Finally, we've also included some content on using core Ray APIs with `Tensorflo
    starting-ray.rst
    actors.rst
    namespaces.rst
-   dependency-management.rst
+   handling-dependencies.rst
```

Comment:
What's the motivation of the name change here?

Reply:
It was suggested here https://github.com/ray-project/ray/pull/19863#discussion_r739037721 @richardliaw is it because "Dependency Management" already has a meaning that's too specific?